test/pool.test.ts

```diff
@@ -522,6 +648,393 @@ describe("EthPool", function () {
     }
     await sender.expectSetReceiverReverts(receiver, 1, "Too many receivers");
   });
+
+  it("Allows sending via a proxy", async function () {
+    const [sender, proxy, receiver] = await getEthPoolUsers();
+    const proxyWeightBase = await sender.pool.PROXY_WEIGHTS_SUM();
+    await proxy.setProxyWeights([[receiver, proxyWeightBase]]);
+    await sender.setProxy(proxy, proxyWeightBase);
+    await sender.setAmountPerBlock(proxyWeightBase);
```

Comment:
Can we choose a different multiple of `proxyWeightBase` here? This would make the relationship between the value and the proxy weight base clearer.

Reply:
Fixed